### PR TITLE
AZP: Upload src.rpm/tar.gz to release page

### PR DESCRIPTION
**Pull request by yshestakov**
* Added `rpm-build` to centos7.Dockerfile
* Changed ref on `centos7:2` container image
  in the azure-pipelines-release.yaml

### CODE REVIEW

File: buildlib/azure-pipelines-release.yml [hunk 1]
```diff
@@ -30,7 +30,22 @@ stages:
...
+          clean: true
+          path: "we/need/to/go/deeper"
+
+        - bash: ./autogen.sh
+          displayName: Setup autotools
+
+        - bash: |
+            set -eE
+            gcc --version
+            ./contrib/configure-release --with-java
```
Comment (yosefe) [inline buildlib/azure-pipelines-release.yml:45]: should be --without-java (need minimal configure for src rpm build)

Comment (yshestakov) [reply buildlib/azure-pipelines-release.yml]: fixed

**Comment by shamisp** [comment]
FYI #5335

**Comment by yshestakov** [comment]
> FYI #5335

I tried to build DEB package manually on this (dev) branch and it worked fine for me:
```
$ dpkg-deb -I ../ucx_1.9.fe983b8_amd64.deb
new Debian package, version 2.0.
size 1021344 bytes: control archive=3264 bytes.
     398 bytes,    11 lines      control
    6114 bytes,    94 lines      md5sums
      82 bytes,     7 lines   *  prerm                #!/bin/sh
     105 bytes,     7 lines      shlibs
      74 bytes,     2 lines      triggers
Package: ucx
Version: 1.9.fe983b8
```
AndreyM/Yossi told me that we need to merge this PR into "master" and back-port into v1.8.1

**Pull request merged**